tests/unit/amo/components/TestLandingPage.js

```diff
@@ -15,27 +18,40 @@ import {
 import I18nProvider from 'core/i18n/Provider';
 import { ErrorHandler } from 'core/errorHandler';
 import { visibleAddonType } from 'core/utils';
-import { dispatchClientMetadata, fakeAddon } from 'tests/unit/amo/helpers';
-import { createStubErrorHandler, fakeI18n } from 'tests/unit/helpers';
+import {
+  createAddonsApiResult,
+  dispatchClientMetadata,
+  fakeAddon,
+} from 'tests/unit/amo/helpers';
+import {
+  createStubErrorHandler,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
 import ErrorList from 'ui/components/ErrorList';
 
 
-describe('<LandingPage />', () => {
-  function renderProps(props = {}) {
+describe(__filename, () => {
+  function renderProps({
+    store = dispatchClientMetadata().store,
+    ...otherProps
+  } = {}) {
     return {
-      dispatch: sinon.stub(),
+      dispatch: store.dispatch,
```

Comment:
You can delete this. It gets set automatically by `mapDispatchToProps()`.

Reply:
actually, that was needed for the `renderAndMount()` method that I did not touch.